sdk/schemaregistry/azure-schemaregistry-client/src/main/module-info.java

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+module com.azure.schemaregistry.client {
+    requires transitive com.azure.core;
+
+    exports com.azure.schemaregistry.client;
+    exports com.azure.schemaregistry.client.rest;
```

Comment:
Does this package have to be exported? It contains codgen rest service client and builder which shouldn't be used anywhere outside this module.

Reply:
Removed rest and rest.models from module-info.